draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1467,8 +1483,10 @@ Constants:
 
 - A and B, the parameters of the Weierstrass curve.
 
-- Z, the smallest (in absolute value) non-square in F such that g(B / (Z *
-  A)) is square in F, breaking ties by choosing the positive value.
+- Z, the unique non-square in F such that g(B / (Z * A)) is square in F
```

Comment:
This definition is correct but difficult to get.
~~~
Z, is a value such that abs(Z) is the smallest in F for which g( B/(Z*A) ) is square. If Z and -Z both meet the above criteria, choose Z such that sgn0(Z) == 1.
~~~

Reply:
Yeah, I struggled with this one a lot.

The issue with the old wording (and I think with the suggested edit) is that it's not that abs(Z) is the smallest, it's that there's no other Z' smaller. In other words, it's `<=`, not `<`, and then we specify a way of breaking the possible tie from `<=`.

Maybe a bulleted list makes it easier? I'll push an edit and let's see how that works.

Reply:
OK, pushed new text using lists. Thoughts?

Reply:
I think it looks better now.